.github/workflows/prepare-release.yml

```diff
@@ -92,11 +92,16 @@ jobs:
           MAVEN_EXEC: ${{ inputs.maven_executable }}
 
       - name: Update latest release version in docs
-        if: ${{ hashFiles('docs/pom.xml') != '' }}
+        if: ${{ hashFiles('docs/pom.xml') != '' || hashFiles('docs/antora.yml') }}
```

Comment:
There are some projects without a pom.xml in the `docs/` module, maybe use `&&`?

Reply:
Or perhaps `git add docs` directly

Reply:
Would you like to adapt it? What I'd need is to add the changed antora.yml. I am far from knowing what different setups other projects use.

Reply:
Sure, I'll do it first thing Monday morning 👍🏻 
Thanks!